src/clr-angular/utils/i18n/common-strings.service.ts

```diff
@@ -30,6 +30,7 @@ export class ClrCommonStringsService implements ClrCommonStrings {
   rowActions = 'Available actions';
   pickColumns = 'Show or hide columns';
   showColumns = 'Show Columns';
+  sortOf = 'Sort of';
```

Comment:
nit: not sure we need the "of". "Sort column" maybe? Any native speakers? :)

Reply:
'Sort' to be